packages/animations/lib/src/open_container.dart

```diff
@@ -167,7 +175,9 @@ class OpenContainer extends StatefulWidget {
   final ShapeBorder openShape;
 
   /// Called when the container was popped and has returned to the closed state.
-  final VoidCallback onClosed;
+  /// The return value from the popped screen is passed to this function as an
+  /// argument.
```

Comment:
note that it may be null if no return value was provided.

Reply:
Should this be included in the documentation?

Reply:
I think it's worth including in the documentation so that the meaning of having a `null` return value isn't ambiguous